projects/vanilla/src/app/home/home.component.ts

```diff
@@ -14,15 +13,14 @@ export class HomeComponent implements OnInit {
   constructor(
     public loginService: LoginService,
     public searchService: SearchService,
-    private titleService: Title,
-    private intlService: IntlService
+    private globalService: GlobalService
   ) {}
 
   /**
    * On initialization set the page title
    */
   ngOnInit() {
-    this.titleService.setTitle(this.intlService.formatMessage("msg#app.name"));
+    this.globalService.setTitle();
```

Comment:
This logic maybe needs to be in the app.component.ts, so the title is also set in the search component ?

Reply:
(or in the app config service)

Reply:
I've setted it first in the config service, but I don't remember why I've moved it here later

Reply:
I remember... When you are in the "seach component" and navigate back to "home" clicking in the "Sineaqu" logo the Title does not change

Reply:
solved this with a TitleStrategy service